Header header
float32 highest
float32 lowest
float32 variance # measure of the spread of a distribution
float64 kurtosis # measure of non-guassianity
float64 skewness
# skewness = 0 : normally distributed.
# skewness > 0 : more weight in the left tail of the distribution.
# skewness < 0 : more weight in the right tail of the distribution. 

